tests/algorithms/test_sqil.py

```diff
@@ -1,25 +1,33 @@
 """Tests `imitation.algorithms.sqil`."""
+from typing import Any, Dict, Optional, Type
 from unittest import mock
 
 import numpy as np
 import pytest
-from stable_baselines3.common import policies, vec_env
+from stable_baselines3 import ddpg, dqn, sac, td3
+from stable_baselines3.common import off_policy_algorithm, policies, vec_env
 from stable_baselines3.common.evaluation import evaluate_policy
 
 from imitation.algorithms import sqil
 from imitation.testing import expert_trajectories, reward_improvement
 
 EXPERT_DATA_TYPES = ["trajectories", "transitions"]
+RL_ALGOS_CONT_ACTIONS = [ddpg.DDPG, sac.SAC, td3.TD3]
 
 
-def get_demos(rng: np.random.Generator, pytestconfig: pytest.Config, data_type):
+def get_demos(
+    env_name: str,
+    rng: np.random.Generator,
+    pytestconfig: pytest.Config,
+    data_type: str,
+):
     cache = pytestconfig.cache
     assert cache is not None
     return expert_trajectories.make_expert_transition_loader(
-        cache_dir=cache.mkdir("experts"),
+        cache_dir=cache.mkdir(env_name.replace("/", "_")),
```

Comment:
Why do we need environment name in the cache directory? Should already be included in the environment path in https://github.com/HumanCompatibleAI/imitation/blob/master/src/imitation/testing/expert_trajectories.py#L74

Reply:
Yeah, indeed, I was confused about the implementation of this function that uses cache and was not sure if I need to make it unique or not. Now I see that this root cache dir can be shared.